File: app/src/main/java/com/donald/musictheoryapp/Music/MusicXML/Time.java
package com.donald.musictheoryapp.Music.MusicXML;

public class Time
{
	public final int beats;
	public final int beatType;

	public Time(int beats, int beatType)
	{
		this.beats = beats;
		this.beatType = beatType;
	}

	public Time clone()
	{
		return new Time(beats, beatType);
	}
}

File: app/src/main/java/com/donald/musictheoryapp/music/MusicXML/Time.kt
package com.donald.musictheoryapp.music.MusicXML

import org.dom4j.Element

class Time(val beats: Int, val beatType: Int) {

    fun addToXml(attributes: Element) {
        attributes.addElement("time").apply {
            addElement("beats").addText(beats.toString())
            addElement("beat-type").addText(beatType.toString())
        }
    }

    fun clone(): Time {
        return Time(beats, beatType)
    }

}